RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,34 +2,60 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
-@{
-    var billId = Guid.Parse(RouteData.Values["guid"].ToString());
-    if (billRepository.GetById(billId) is null)
-    {
-        <div id="errorText" style="color:red"> Bill was not found!</div>
-    }
-    else
-    {
-        var item = billRepository.GetById(billId);
-        <div class="bill-summary">
-            <div t-id="description">@item.Description</div>
-            <div t-id="personWhoPaid">Paid by: @(item.EntryLines[0].Person.FirstName)
-                @(item.EntryLines[0].Person.LastName) </div>
-            <div t-id="date">@item.Date.ToString("D")</div>
-            <div t-id="amount">Total amount: @item.EntryLines[0].Amount€</div>
-        </div>
-        <div class="bill">
-            @for (int i = 1; i < item.EntryLines.Length; i++)
-            {
-                <div t-id="entryLine">
-                    <div t-id="person">@(item.EntryLines[i].Person.FirstName)
-                        @(item.EntryLines[i].Person.LastName): </div>
-                    <div t-id="amountPerPerson">@(-item.EntryLines[i].Amount)€ </div>
+<head>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
+
+<div class="container mt-3">
+    @{
+        var billId = Guid.Parse(RouteData.Values["guid"].ToString());
+        if (billRepository.GetById(billId) is null)
+        {
+            <div class="alert alert-danger text-center">
+                Bill was not found!
+            </div>
+        }
+        else
+        {
+            <h1 class="text-center mb-4">Bill Details</h1>
+            var item = billRepository.GetById(billId);
+            <div class="p-3 border rounded mx-auto" style="max-width: 900px;">
```

Comment:
The `max-width` should be applied to the container and not all elements inside of the container. Or as an alternative use columns to align the box in the middle of the content. But in this case we probably want a smaller container as that will affect all the contained elements.